scripts/test-with-server.sh

```
@@ -0,0 +1,30 @@
+
+# サーバーが起動するまで待つ
+echo "サーバーの起動を待っています..."
+sleep 10
+
+# サーバーが起動したか確認
+if curl -s http://localhost:3000 > /dev/null; then
+    echo "サーバーが起動しました"
+    
+    # テストを実行
+    echo "テストを実行しています..."
+    npx playwright test $@
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)